src/containers/ChartView/index.tsx

```diff
@@ -26,10 +26,46 @@ import Typography from '@mui/material/Typography';
 import ChartSlider from 'components/ChartSlider';
 import { getCountryName } from 'utils/helperFunctions';
 import { useTheme } from '@mui/material/styles';
-
 import { ChartContainer } from './styled';
+import { ToggleButton, ToggleButtonGroup } from '@mui/material';
+
+type ChartTypeValues = {
+    title: string;
+    dataKey: string;
+    tooltipName: string;
+};
+
+enum chartTypeNames {
```

Comment:
`enum` should be named with first capital letter the same as types and interfaces